aurpublish.in

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+AUR_GIT_URI='ssh://aur@aur.archlinux.org'
```

Comment:
In general I prefer the `aur@aur.archlinux.org:` style of ssh usage over the use of `ssh://`.

Reply:
Fair enough. I have changed the default to use scp-like syntax in commit 1e8e44d.